contracts/Delegator.sol

```diff
@@ -9,6 +9,7 @@ contract Delegator is ACL {
     IAssetManager public assetManager;
 
     function upgradeDelegate(address newDelegateAddress) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        require(newDelegateAddress != address(0x0), "Zero Address check");
```

Comment:
Please check if there is no difference in address(0x0) and address(0), as in OZ, they are using address(0)

Reply:
there is no difference, we can use either 0x0 or 0